### PR TITLE
azure-keyvault meta-package

### PR DESCRIPTION
**Pull request by lmazuel**
Fix https://github.com/Azure/azure-sdk-for-python/issues/7954

### CODE REVIEW

File: sdk/keyvault/azure-keyvault/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Azure Key Vault client libraries for Python
+
+This is the Microsoft Azure Key Vault libraries bundle.
+
+This package does not contain any code in itself. It installs a set
+of packages that provide APIs for Key Vault operations:
+
+- [azure-keyvault-keys v4.0.0b5](https://pypi.python.org/pypi/azure-keyvault-keys>)
+- [azure-keyvault-secrets v4.0.0b5](https://pypi.python.org/pypi/azure-keyvault-secrets>)
+- [azure-keyvault-certificates v4.0.0b5](https://pypi.python.org/pypi/azure-keyvault-certificates>)
```
Comment (chlowell) [inline sdk/keyvault/azure-keyvault/README.md]: Trailing `>` on these URLs

File: sdk/keyvault/azure-keyvault/setup.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    author='Microsoft Corporation',
+    author_email="azurekeyvault@microsoft.com",
+    url="https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-keyvault",
+    classifiers=[
+        'Development Status :: 5 - Production/Stable',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.4',
```
Comment (chlowell) [inline sdk/keyvault/azure-keyvault/setup.py]: I'd strike 3.4 from the list.

File: sdk/keyvault/azure-keyvault/setup.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    version='4.0.0b5',
+    description='Microsoft Azure Resource Management Client Libraries for Python',
+    long_description=README + "\n\n" + HISTORY,
+    long_description_content_type="text/markdown",
+    license='MIT License',
+    author='Microsoft Corporation',
+    author_email="azurekeyvault@microsoft.com",
+    url="https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-keyvault",
+    classifiers=[
+        'Development Status :: 5 - Production/Stable',
```
Comment (chlowell) [inline sdk/keyvault/azure-keyvault/setup.py]: It's stable in a sense but docs and version suggest it's prerelease.

File: sdk/keyvault/azure-keyvault/setup.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
```
Comment (chlowell) [inline sdk/keyvault/azure-keyvault/setup.py]: Exciting! 🎂

File: .docsettings.yml [hunk 5]
```diff
@@ -2,9 +2,8 @@ omitted_paths:
   - tools/*
   - scripts/*
   - azure-mgmt-netapp/tests/*
-  - "azure-mgmt*/*"
```
Comment (johanste) [inline .docsettings.yml]: I assume these entries were intentionally removed?

File: sdk/keyvault/azure-keyvault/setup.py [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+with open("README.md", encoding="utf-8") as f:
+    README = f.read()
+with open("HISTORY.md", encoding="utf-8") as f:
+    HISTORY = f.read()
+
+setup(
+    name='azure-keyvault',
+    version='4.0.0b5',
+    description='Microsoft Azure Resource Management Client Libraries for Python',
```
Comment (johanste) [inline sdk/keyvault/azure-keyvault/setup.py]: `description` should be updated to indicate that these are the KeyVault client libraries...

File: .docsettings.yml (hunk 5 above)
Comment (lmazuel) [reply .docsettings.yml]: yes, there is no packages at the root of the repo, that's legacy from when we were transitioning into "sdk" folder

File: sdk/keyvault/azure-keyvault/setup.py (hunk 6 above)
Comment (lmazuel) [reply sdk/keyvault/azure-keyvault/setup.py]: fixed

File: sdk/keyvault/azure-keyvault/setup.py [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+from setuptools import setup
+
+with open("README.md", encoding="utf-8") as f:
+    README = f.read()
+with open("HISTORY.md", encoding="utf-8") as f:
+    HISTORY = f.read()
+
+setup(
+    name='azure-keyvault',
+    version='4.0.0b5',
```
Comment (johanste) [inline sdk/keyvault/azure-keyvault/setup.py]: Version number?

Comment (lmazuel) [reply sdk/keyvault/azure-keyvault/setup.py]: fixed

File: sdk/keyvault/azure-keyvault/setup.py [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+setup(
+    name='azure-keyvault',
+    version='4.0.0',
+    description='Microsoft Azure Key Vault Client Libraries for Python',
+    long_description=README + "\n\n" + HISTORY,
+    long_description_content_type="text/markdown",
+    license='MIT License',
+    author='Microsoft Corporation',
+    author_email="azurekeyvault@microsoft.com",
```
Comment (johanste) [inline sdk/keyvault/azure-keyvault/setup.py]: This should be azuresdk. no? Not blocking the release.

**Pull request merged**